doc/wiki/source_code/Github.md

```diff
@@ -18,6 +18,58 @@ git clone https://github.com/pmodels/mpich.git (for non-core developers wi
 git clone git@github.com:pmodels/mpich.git (for core developers with commit rights)
 ```
 
+### Dependencies and Submodules
+
```

Comment:
Refer to earlier comment, I see the removed content is added in this commit. Content move need be contained in a single commit rather than one commit remove and another commit add

Reply:
This content was properly done.